src/docs/sdk/research/performance/index.mdx

```diff
@@ -192,6 +192,160 @@ In the example above, if an error bubbles up the call stack we want to be able t
+
+Because all data must go in a transaction. Sentry's browser SDK creates a transaction for every page load and every navigation. Those transactions must end at some time.
+
+If the browser tab is closed before the transaction is finished and sent to Sentry, all collected data is lost. Therefore, the SDK needs to balance the risk of losing all data and capturing as much useful data as possible.
+
+Transactions are finished after a certain idle time after the last activity has been observed (for example, outgoing HTTP requests). This means that the duration of the page load / navigation transaction is a rather arbitrary value that can't necessarily be compared or improved, as it doesn't indicate any concrete and undestandable duration.
+
+We counter the limitation by focusing on the LCP Web Vital as the default performance metric for browser, but, as discussed above, the final LCP value might be lost.
+
+### In-memory Buffering Affects Servers
+
+Any system that may operate with several concurrent transactions at one time, which is the case of web servers, will be required to buffer complete span trees in memory.
+
+This means recording 100% of spans and 100% of transactions for many server-side applications, even in a simplified form, is not feasible due to the overhead incurred.
+
+### Lack of Batch Ingestion
+
+When multiple concurrent transactions finish nearly at the same time, SDKs must send as many requests as there are transactions. There is no provision to batching multiple transactions into a single request.
+
+This inneficiency means at least extra bandwidth consumption for frontends (browser, mobile) and extra overhead for backends.
+
+### Compatibility
+
+The special treatment of transactions is incompatible with OpenTelemetry which means we cannot implement an OpenTelemetry Exporter that can feed data into Sentry (though we have a [Sentry Exporter with a major correctness limitation](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/sentryexporter#known-limitations)). Likewise we cannot leverage OpenTelemetry SDKs and instrumentations.
+
+## Summary
+
+We have learned a lot with the current tracing implementation at Sentry. This document is an attempt to capture many of the known limitations to serve as basis for future improvement.
+
+Tracing is a complex subject, and taming that complexity is no easy feat.
+
+The first group of issues related to **scope propagation** is a concern exclusive to SDKs and how they are designed. Addressing that will require internal architecture changes to all SDKs, including the redesign of old features like breadcrumbs. Those would be pre-requisites to implementing simple-to-use tracing helpers like a `trace` function that works in any context and captures accurate and reliable performance data.
+
+Note that such changes would almost necessarily mean new major versions of SDKs that break compatibility with existing versions.
```

Comment:
I made the issues in the group - rather than the group itself - the subject of the first sentence, to avoid the correct-but-awkward-sounding "single group, plural issues, single is" construction.

I also felt like the "Note that..." sentence had a hard time standing on its own as a full paragraph. I'm guessing you did that to emphasize the point, though, so I italicized it.

```suggestion
Issues in the first group - those related to **scope propagation** - are a concern exclusive to SDKs and how they are designed. Addressing them will require internal architecture changes to all SDKs, including the redesign of old features like breadcrumbs, but making such changes is a prerequisite for implementing simple-to-use tracing helpers like a `trace` function that works in any context and captures accurate and reliable performance data. Note that such changes _would almost certainly mean releasing new major versions of SDKs that break compatibility with existing versions_.
```

Reply:
Took the suggestion without the italics in the last sentence.